spec/locale.html

```diff
@@ -452,5 +550,265 @@ <h1>
+      </emu-note>
+    </emu-clause>
+
+    <emu-clause id="sec-weekday-to-string" type="abstract operation">
+      <h1>
+        WeekdayToString (
+          _fw_: a String,
+        ): a String
+      </h1>
+      <dl class="header">
+      </dl>
+      <emu-alg>
+        1. For each row of <emu-xref href="#table-locale-first-day-option-value"></emu-xref>, except the header row, in table order, do
+          1. Let _w_ be the name given in the Weekday column of the current row.
+          1. Let _s_ be the name given in the String column of the current row.
+          1. If _fw_ is equal to _w_, return _s_.
+        1. Return _fw_.
+      </emu-alg>
+
+      <emu-table id="table-locale-first-day-option-value">
+        <emu-caption>First <emu-not-ref>Day</emu-not-ref> String and Value</emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th>Weekday</th>
+              <th>String</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tr>
+            <td>*"0"*</td>
+            <td>*"sun"*</td>
+            <td>*7*<sub>𝔽</sub></td>
+          </tr>
+          <tr>
+            <td>*"1"*</td>
+            <td>*"mon"*</td>
+            <td>*1*<sub>𝔽</sub></td>
+          </tr>
+          <tr>
+            <td>*"2"*</td>
+            <td>*"tue"*</td>
+            <td>*2*<sub>𝔽</sub></td>
+          </tr>
+          <tr>
+            <td>*"3"*</td>
+            <td>*"wed"*</td>
+            <td>*3*<sub>𝔽</sub></td>
+          </tr>
+          <tr>
+            <td>*"4"*</td>
+            <td>*"thu"*</td>
+            <td>*4*<sub>𝔽</sub></td>
+          </tr>
+          <tr>
+            <td>*"5"*</td>
+            <td>*"fri"*</td>
+            <td>*5*<sub>𝔽</sub></td>
+          </tr>
+          <tr>
+            <td>*"6"*</td>
+            <td>*"sat"*</td>
+            <td>*6*<sub>𝔽</sub></td>
+          </tr>
+          <tr>
+            <td>*"7"*</td>
+            <td>*"sun"*</td>
+            <td>*7*<sub>𝔽</sub></td>
+          </tr>
+        </table>
+      </emu-table>
+    </emu-clause>
+
+    <emu-clause id="sec-string-to-weekday-value" type="abstract operation">
+      <h1>
+        StringToWeekdayValue (
+          _fw_: a String,
+        ): an integral Number or *undefined*
+      </h1>
+      <dl class="header">
+      </dl>
```

Comment:
Please rename for specificity and add a description, e.g.
```suggestion
      <h1>
        FirstDayOfWeekUValueToNumber (
          _fw_: a String,
        ): an integral Number or *undefined*
      </h1>
      <dl class="header">
        <dt>description</dt>
        <dd>It converts a <a href="https://unicode.org/reports/tr35/#UnicodeFirstDayIdentifier">Unicode First Day Identifier</a> defined in <a href="https://unicode.org/reports/tr35/#Key_And_Type_Definitions_">Unicode Technical Standard #35 Part 1 Core Section 3.6.1 Key and Type Definitions</a> to a numeric ISO 8601 <em>calendar day of week</em>, and returns any other string unmodified.</dd>
      </dl>
```